lib/susedistribution.pm

```diff
@@ -216,6 +218,7 @@ sub x11_start_program {
     }
     # asserting program came up properly
     die "Did not find target needle for tag(s) '@target'" if match_has_tag('desktop-runner-border') || match_has_tag('desktop-runner-plasma-suggestions');
+    $self->{in_x11_start_program} = 0;
```

Comment:
> An alternative might be to set a test variable instead of a member variable, e.g. set_var('IN_X11_START_PROGRAM', 1)


I guess that's the most feasible solution